### PR TITLE
Just put the version as string

### PR DESCRIPTION
**Pull request by achanda**

### CODE REVIEW

**Comment by dadgar** [comment]
Lets add a unit test. The regex currently there is incorrect: http://play.golang.org/p/asD0nphBwe

**Comment by dadgar** [comment]
re := regexp.MustCompile(`rkt version (\d[\.\d]+)`)

Should work

**Comment by achanda** [comment]
Thanks @dadgar

File: client/driver/rkt.go [hunk 1]
```diff
@@ -21,8 +21,8 @@ import (
 )
 
 var (
-	reRktVersion  = regexp.MustCompile("rkt version ([\\d\\.]+).+")
-	reAppcVersion = regexp.MustCompile("appc version ([\\d\\.]+).+")
+	reRktVersion  = regexp.MustCompile("rkt version (\\d[.\\d]+)")
+	reAppcVersion = regexp.MustCompile("appc version (\\d[.\\d]+)")
```
Comment (dadgar) [inline client/driver/rkt.go]: Can you change the it from quotes to backticks. That way you don't have to double delimit the backslashes.

**Comment by dadgar** [comment]
Looks good! Minor feedback

**Comment by achanda** [comment]
Cool! I didn't know Go has back tick strings.

**Comment by dadgar** [comment]
Thanks!

**Pull request merged**